EIPS/eip-multi_resource_non_fungible_token_standard.md

```diff
@@ -0,0 +1,366 @@
+---
+eip: <to be assigned>
+title: Multi-Resource Token
+description: An interface for Multi-Resource tokens.
+author: Bruno Škvorc (@Swader), Cicada (@CicadaNCR), Steven Pineda (@steven2308), Stevan Bogosavljevic (@stevyhacker), Jan Turk (@ThunderDeliverer)
+discussions-to: <URL>
```

Comment:
Please don't forget to create a discussion thread :)

Reply:
I intended to add it at the same time as the EIP number was assigned (wanted to include the EIP number in the discussion title). I'll open a discussion now and add it :)

Reply:
Done ✅

Reply:
> I intended to add it at the same time as the EIP number was assigned (wanted to include the EIP number in the discussion title). I'll open a discussion now and add it :)

I thought the bot was going to assign the number and create the thread automatically.

Reply:
> > I intended to add it at the same time as the EIP number was assigned (wanted to include the EIP number in the discussion title). I'll open a discussion now and add it :)
> 
> I thought the bot was going to assign the number and create the thread automatically.

It only reports the missing EIP number and that an editor should assign it. Should I just use the PR number? So 5773?

Reply:
I'll use it as this EIP is already being referred to as EIP-5773:
https://weekinethereumnews.com/week-in-ethereum-news-october-15-2022
https://twitter.com/WeekInEthNews/status/1582456945750904832?s=20&t=0H712tazBBqHtRQgD227OA

The number can be changed at any time, if you decide to assign a different one 🤔

Reply:
> I thought the bot was going to assign the number and create the thread automatically.

Not yet. We're waiting on #5508.